Python/Product/BuildTasks/Microsoft.PythonTools.Web.targets

```diff
@@ -94,30 +93,23 @@ permissions and limitations under the License.
     </ItemGroup>
   </Target>
 
+  <Target Name="CollectFilesFromIntermediateAssembly" />
```

Comment:
It's actually suppressing the "standard" target by the same name, which tries to get the compiled DLL and its debugging symbols.